hack/testing/util.sh

```diff
@@ -378,15 +388,28 @@ function curl_es_pod_with_token_and_input() {
     local args=( "${@:-}" )
     local authtoken="-H Authorization: Bearer $test_token"
     local contenttype="-H Content-type:application/json"
-    local xff="-H X-Forwarded-For:127.0.0.1"
```

Comment:
why are we removing this if we test by curling from within the elasticsearch container? does the proxy somehow pick this request up?

Reply:
the calling test will set the header. This was added with multi-tenancy test case when es-proxy was not there. But now we have es-proxy, and multi-tenancy test case will go through es-proxy, we shouldnt add it from utils function.